app/src/server/tasks/worker.ts

```diff
@@ -72,17 +72,17 @@ const runner = await run({
     {
       task: generateInvoices.task.identifier,
       // run at 2 AM UTC on the first day of each month
-      pattern: "0 1 * * *", // TODO: Set "0 2 1 * *". This is a temp change to test in production. It runs at 1 am every day.
+      pattern: "* 2 1 * *",
       identifier: generateInvoices.task.identifier,
     },
 
-    // TODO: Uncomment this when we are ready to charge invoices
-    // {
-    //   task: chargeInvoices.task.identifier,
-    //   // run at 8 AM UTC, on the first day of each month, after invoices are created
-    //   pattern: "0 8 1 * *",
-    //   identifier: chargeInvoices.task.identifier,
-    // },
+    //If an invoice payment failed, it will retry on the next day
+    {
+      task: chargeInvoices.task.identifier,
```

Comment:
If we're charging invoices immediately do we still need this task?

Reply:
This task loops through pending invoices and tries to charge them again. In most cases, it means that the payment failed initially; a user received a notification and updated a payment.
We could charge each time a user updates a payment method, but there are other cases, like when a user does not have sufficient funds on a card or some banking errors. In such cases, we need to run a task that charges again.
The only question is how often we should run it.

I just added a logic that searches for pending invoices not older than 3 months. If we could not charge an invoice for 3 months, we should probably stop trying.